src/device/mod.rs

```diff
@@ -1,18 +1,40 @@
+use std;
+
 use manager;
 use manager::NetworkManager;
 
 
 #[derive(Debug)]
 pub struct Device {
-    interface: String,
-    path: String,
-    device_type: DeviceType,
-    state: DeviceState,
-    real: bool,
+    pub interface: String,
+    pub path: String,
+    pub device_type: DeviceType,
+    pub state: DeviceState,
+    pub real: bool,
```

Comment:
We do not use this one indeed, I just removed it. I added it when we did the `list` function thinking that we might need it, but we don't.